docs/Software Architecture Document/Components View/index.md

```diff
@@ -148,6 +148,21 @@ The view is represented by module and subsystem diagrams that show the system's
     - Not work for Android apps.
     - Not work for apps connected via SDL protocol version lower than 4.
 
+#### Plugin Manager
+  - *Responsibility*
+    - Loads all .so files from specific directory, checking if they’re exporting required methods
```

Comment:
@LuxoftAKutsan, but it's better to use something like "Dynamic library" and it helps to avoid changes in future =)